app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/EventDetailFragment.java

```diff
@@ -156,7 +159,7 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             if (lecture != null && lecture.dateUTC > 0) {
                 DateFormat df = SimpleDateFormat
                         .getDateTimeInstance(SimpleDateFormat.SHORT, SimpleDateFormat.SHORT);
-                t.setText(df.format(new Date(lecture.dateUTC)) + " - " + room);
+                t.setText(df.format(new Date(lecture.dateUTC)) + " " + eventDuration + " min. - " + room);
```

Comment:
Lint warning: Please do the String concatenation beforehand. Use a local variable.